packages/analytics-js/src/components/userSessionManager/UserSessionManager.ts

```diff
@@ -352,17 +352,28 @@ class UserSessionManager implements IUserSessionManager {
       const curStore = this.storeManager?.getStore(
         storageClientDataStoreNameMap[storageType] as string,
       );
-      const key = entries[sessionKey]?.key as string;
-      if (value && (isString(value) || isNonEmptyObject(value))) {
-        // if useServerSideCookies load option is set to true
-        // set the cookie from server side
-        if (state.loadOptions.value.useServerSideCookies && storageType === COOKIE_STORAGE) {
-          this.setServerSideCookie([{ name: key, value }], curStore);
+      if (curStore) {
```

Comment:
This `if` condition may not be required as we're already using the optional chaining operator while using it.
Also, it shouldn't discourage us from executing the server-side cookies.

Reply:
Addressed